epsagon/constants.py

```diff
@@ -56,3 +56,4 @@ def is_strong_key(key):
 
 
 STEP_DICT_NAME = 'Epsagon'
+TRACE_ID_DICT_NAME = '_epsagon_trace_id'
```

Comment:
This name is not good. It's no the trace ID and it's not a dict.
Maybe just call it `epsagon_event_id`?